File: Python/product.py
# Kelas Product
class Product():
    __price = 0
    __idProduct = ""

    # Constructor
    def __init__(self, price = 0, idProduct = ""):
        self.__price = price
        self.__idProduct = idProduct

    # Setter
    def setPrice(self, price):
        self.__price = price

    def setIdProduct(self, idProduct):
        self.__idProduct = idProduct

    # Getter
    def getPrice(self):
        return self.__price
    
    def getIdProduct(self):
        return self.__idProduct

File: PHP/product.php
<?php

// Kelas Product
class Product
{
    private $price = 0;
    private $idProduct = "";

    // Constructor
    public function __construct($price = 0, $idProduct = "")
    {
        $this->price = $price;
        $this->idProduct = $idProduct;
    }

    // Setter
    public function setPrice($price)
    {
        $this->price = $price;
    }

    public function setIdProduct($idProduct)
    {
        $this->idProduct = $idProduct;
    }

    // Getter
    public function getPrice()
    {
        return $this->price;
    }

    public function getIdProduct()
    {
        return $this->idProduct;
    }
}

?>